### DISCUSSION TITLE
Stacked Chart - Dark Mode -  default line color is [black] invisible

### DISCUSSION THREAD

**Discussion by jbenzing** [community/community#38184] [Accessibility] [answered]
When grouping has more than 8 items, the remaining items revert to black in dark mode.

This makes the lines invisible. 

Can more colors be added, or at least change the default to gray?

![image](https://user-images.githubusercontent.com/35874616/199762619-502fc8ed-be07-4504-a519-13f0747e2709.png)

**Comment by ossanna16** [comment]
Hi there @jbenzing and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jbenzing** [comment, marked as answer]
This issue has been fixed.  More colors were added.

**Comment by cehfisher** [comment]
Hi @jbenzing - this issue appears to be resolved [per this comment](https://github.com/orgs/community/discussions/38184#discussioncomment-4856639). Could you please close this issue? Thank you! (cc @queenofcorgis)
